app/src/main/res/values/styles.xml

```diff
@@ -33,7 +33,7 @@
 
 
     <!-- Theme for the QR-Code scanner activity. -->
-    <style name="AppOverlayTheme" parent="@style/Theme.AppCompat.Light">
+    <style name="AppOverlayTheme" parent="@style/Theme.AppCompat">
```

Comment:
@michaelWuensch The light theme converted the text in the checkbox to black on my site.